natgeo/mortar
# [SYTO-1045] Video Playlist Design Tweaks

https://jira.natgeo.com/browse/SYTO-1045

#### How to test
1. ``npm run gulp serve``
2. Go to components and scroll down until Video Playlist cmp.
3. Check all the items in the acceptance criteria.
